simplyblock_web/blueprints/snode_ops_k8s.py

```diff
@@ -358,6 +358,29 @@ def spdk_process_start(body: SPDKParams):
             'PCI_ALLOWED': ssd_pcie_list,
             'TOTAL_HP': total_mem_mib
         }
+
+        job_template = env.get_template('storage_init_job.yaml.j2')
+        job_yaml = yaml.safe_load(job_template.render(values))
+        batch_v1 = core_utils.get_k8s_batch_client()
+        job_resp = batch_v1.create_namespaced_job(namespace=namespace, body=job_yaml)
+        msg = f"Job created: '{job_resp.metadata.name}' in namespace '{namespace}"
+        logger.info(msg)
+
+        node_utils_k8s.wait_for_job_completion(job_resp.metadata.name, namespace)
+        logger.info(f"Job '{job_resp.metadata.name}' completed successfully")
+
+        batch_v1.delete_namespaced_job(
```

Comment:
Maybe we could leave the job as it is without deleting it. That way the users could check the logs